tests/test_poe.py

```diff
@@ -31,3 +33,31 @@ def test_data_gatherer_needed(self):
             poe.data_gatherer_needed({}),
             set(['cinder', 'nova', 'neutron']),
         )
+
+    @mock.patch('prometheus_openstack_exporter.config')
+    def test_get_nova_info(self, config):
+        config.return_value = {}
+        prodstack = {'tenants': []}
+        nova = mock.Mock()
+        nova.aggregates = mock.MagicMock()
+        nova.flavors = mock.MagicMock()
+        nova.hypervisors = mock.MagicMock()
+        nova.servers = mock.MagicMock()
+        nova.servers.list = mock.MagicMock()
+        nova.services = mock.MagicMock()
+
+        data_gatherer = poe.DataGatherer()
+        data_gatherer._get_nova_info(nova, None, prodstack)
+
+        expected = [
+            mock.call(search_opts={'marker': '', 'limit': '100', 'status': 'ACTIVE', 'all_tenants': '1'}),
+            mock.call(search_opts={'marker': '', 'limit': '100', 'status': 'SHUTOFF', 'all_tenants': '1'}),
+            mock.call(search_opts={'marker': '', 'limit': '100', 'status': 'ERROR', 'all_tenants': '1'}),
+            mock.call(search_opts={'marker': '', 'limit': '100', 'status': 'VERIFY_RESIZE', 'all_tenants': '1'}),
+            mock.call(search_opts={'marker': '', 'limit': '100', 'status': 'SHELVED_OFFLOADED', 'all_tenants': '1'}),
+            mock.call(search_opts={'marker': '', 'limit': '100', 'status': 'SUSPENDED', 'all_tenants': '1'}),
```

Comment:
Can we sort these alphabetically by status as well?

Reply:
Fixed.